packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market.mligo

```diff
@@ -79,7 +79,7 @@ let buy_token(sale, storage: sale_param * storage) : (operation list * storage)
   | Some s -> s
   in
   let tx_money_op1 = transfer_money(sale.tokens.money_token_address, sale.tokens.money_token_token_id, sale_price, Tezos.sender, Tezos.self_address) in
-  let tx_money_op2 = transfer_money(sale.tokens.money_token_address, sale.tokens.money_token_token_id, sale_price, Tezos.self_address, sale.sale_seller) in
+  let tx_money_op2 = transfer_money(sale.tokens.money_token_address, sale.tokens.money_token_token_id, sale_price, Tezos.self_address, sale.seller) in
```

Comment:
Why not just transfer money directly from buyer to seller bypassing marketplace contract?